hack/.golint_failures

```diff
@@ -336,6 +336,7 @@ pkg/registry/storage/rest
 pkg/registry/storage/storageclass
 pkg/registry/storage/storageclass/storage
 pkg/routes
+pkg/scheduler/apis/config/v1alpha1
```

Comment:
can we fix the lint issues?

Reply:
I don't think so, as it's always gonna complain about `SetDefaults_Foo`, right? (no underscores allowed)

Reply:
> as it's always gonna complain about SetDefaults_Foo, right? (no underscores allowed)

@luxas Correct.

golint will complain

> don't use underscores in Go names; func SetDefaults_KubeSchedulerConfiguration should be SetDefaultsKubeSchedulerConfiguration